### PR TITLE
✨  [Story video] Create service to build the cache url

### PR DESCRIPTION
**Pull request by mszylkowski**
Creates the `amp-cache-url` extension that installs the service for `amp-toolbox-cache-url`

### CODE REVIEW

File: extensions/amp-cache-url/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Google video cache for amp-video
```
Comment (gmajoulet) [inline extensions/amp-cache-url/README.md:1]: Please update the readme

File: extensions/amp-cache-url/0.1/amp-cache-url.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export class AmpCacheUrlService {
+  /**
+   * Create video cache service
```
Comment (gmajoulet) [inline extensions/amp-cache-url/0.1/amp-cache-url.js:19]: Please update

File: examples/amp-story/videos-google-cache.html [hunk 3]
```diff
@@ -0,0 +1,35 @@
+<!doctype html>
```
Comment (gmajoulet) [inline examples/amp-story/videos-google-cache.html:1]: Please remove

**Comment by gmajoulet** [comment]
Can you also update the PR's name?

File: extensions/amp-cache-url/README.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# Cache URL service
+
+This extension installs a service to convert URLs to cache URLs.
```
Comment (gmajoulet) [inline extensions/amp-cache-url/README.md:3]: Please link to https://github.com/ampproject/amp-toolbox/tree/main/packages/cache-url

File: examples/amp-story/videos-google-cache.html [hunk 5]
```diff
@@ -0,0 +1,34 @@
+<html ⚡ lang="en">
```
Comment (gmajoulet) [inline examples/amp-story/videos-google-cache.html:1]: Please remove

**Review by gmajoulet — APPROVED** [review]

**Comment by gmajoulet** [comment]
cc @danielrozenberg  for owners approval

image

**Review by danielrozenberg — APPROVED** [review]
Approval for build-system/*

**Review by gmajoulet — APPROVED** [review]

**Pull request merged**